modules/support.adoc

```diff
@@ -20,7 +20,7 @@ ifdef::openshift-enterprise,openshift-dedicated[]
 When submitting a support case, it is recommended to provide the following information about your cluster to Red Hat Support to aid in troubleshooting:
 
 * Data gathered using the `oc adm must-gather` command
-* The unique cluster ID
+* The unique cluster ID. (This value is autofilled when you navigate to *(?) Help* -> *Open Support Case* or *Report Bug*.)
```

Comment:
This 'additonal text' should probably be in a call out or not.

Reply:
I agree, though lines 20-23 are already in an `important note` callout. Also, this note is already conditionalized for Enterprise and Dedicated, so I removed the `Report Bug` ref.